src/hooks/useTableColumns.js

```diff
@@ -130,6 +130,19 @@ export const useTableColumns = () => {
 
   const individualContributionsColumns = useMemo(
     () => [
+      {
+        Header: 'Donation Date',
+        id: 'date_occurred',
+        accessor: (r) => {
+          const d = new Date(r.date_occurred)
+          return d.toLocaleDateString('en-US', {
+            year: 'numeric',
+            month: 'short',
+            day: '2-digit',
+          })
+        },
```

Comment:
this is _really_ nitpicky of me, and you don't need to implement this at all: it's more a trick/preference thing, _but_ you could do this all in one line like so:

```suggestion
        accessor: (r) => new Date(r.date_occurred).toLocaleDateString('en-US', {
            year: 'numeric',
            month: 'short',
            day: '2-digit',
          }),
```

if your function can return in one line, you don't need curly brackets, or even a `return`. Just whatever that line returns is what your function returns